### PR TITLE
HPCC-13170 Stats Viz Issues

### PR DESCRIPTION
**Pull request by GordonSmith**
Double Click Sunburst Caused screen glitches.
Unable to zoom out of scopes once you double clicked.
Pie charts should show aggregate of "Kind" rather than stat count.

Fixes HPCC-13170

Signed-off-by: Gordon Smith gordon.smith@lexisnexis.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

File: esp/src/Visualization [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit da4fdbc3eedec680f916d71c3f2d414730185a8b
+Subproject commit a9ab832b5106f8b2b5073e8b0c9b6dcf5bd6cf38
```
Comment (richardkchapman) [inline esp/src/Visualization]: Is this a tagged version?

Comment (GordonSmith) [inline esp/src/Visualization]: Yes - https://github.com/hpcc-systems/Visualization/releases/tag/HPCC-13170

**Comment by miguelvazq** [comment]
Looks fine.

**Pull request merged**